src/main/resources/com/puppycrawl/tools/checkstyle/checks/coding/messages_fr.properties

```diff
@@ -35,7 +35,7 @@ missing.ctor=Il manque un constructeur à la classe.
 missing.package.declaration=Déclaration de package manquante.
 missing.super.call=La méthode ''{0}'' devrait appeler ''super.{0}''.
 missing.switch.default=Il manque le cas "default" dans le bloc "switch".
-missing.switch.nullcase=Switch utilisant des types de r�f�rence doit avoir un cas nul.
+missing.switch.nullcase=Switch utilisant des types de r f rence doit avoir un cas nul.
```

Comment:
Unrelated messes are affected.
Unicode symbols are replaced to space, not good.
There are few updates like this

Reply:
Weird, fixed